docs/HowTo/Configure/Configure-HA/Sample-Configuration.md

```diff
@@ -0,0 +1,94 @@
+description: Sample Load Balancers 
+<!--- END of page meta data -->
+
+# Sample Load Balancer Configurations 
+
+## AWS 
+
+For AWS, we recommend the Classic Load Balancer. The Classic Load Balancer routes layer 4 and 7. Register
+the Hyperledger Besu instances to the load balancer and use the [liveness endpoint](../../Interact/APIs/Using-JSON-RPC-API.md#readiness-and-liveness-endpoints)
+for health checks. 
+
+Another alterntive is the Application Load Balancer:
+ 
+* Configure 1 target group with n nodes 
+* Use the [liveness endpoint](../../Interact/APIs/Using-JSON-RPC-API.md#readiness-and-liveness-endpoints) for health checks
+* Register the Besu instances multiple times with different ports. Similar to configuring microservices 
+on Elastic Container Service (ECS) or Elastic Kubernetes Service (EKS) 
+* Configure multiple listeners with one per port you're using and route to the target group. 
+
+## Elastic Kubernetes Service  
+
+For Elastic Kubernetes Service (AWS Kubernetes service) use the same load balancer configuration as when
+running nodes in Kubernetes. Use labels to specify nodes for the load balanced group. 
+
+## Manual Configurations 
+
+Where applicable, we strongly recommend using service discovery. That is, pair your load balancer 
+configuration with something that dynamically detects new nodes and failures. 
```

Comment:
Rename `failures` to `removed failed nodes`

Reply:
Done